src/main/app/case/from-api-format.ts

```diff
@@ -97,6 +97,12 @@ const fields: FromApiConverters = {
   dueDate: data => ({
     dueDate: dayjs(data.dueDate).format('D MMMM YYYY'),
   }),
+  coDateAndTimeOfHearing: data => ({
+    coDateAndTimeOfHearing: dayjs(data.coDateAndTimeOfHearing).format('D MMMM YYYY'),
```

Comment:
i split this into two variables as this value has both date and time. [pr](https://github.com/hmcts/nfdiv-frontend/pull/1631/files#diff-f11587349bf8795d977c6abbc6f3877355a7f79117b38a04085342e17aaeac35R114).

i guess its fine for you as you dont need it and i can just update it in my pr.

Reply:
it worked just formatting it like this, is the time of the hearing something you need to retain?

Reply:
ah just saw it is on your pr, probably easier if you just update it with your pr after